zeppelin-server/pom.xml

```diff
@@ -457,9 +457,6 @@
   <profiles>
     <profile>
       <id>using-source-tree</id>
-      <activation>
```

Comment:
This is also needed. Please revert this change.

Reply:
Yes. Instead of deleting the line, I changed the activeByDefulat value to false.

Reply:
Yes. Instead of deleting the line, I changed the activeByDefulat value to false.

Reply:
Yes, that should be reasonable but I think it's better not to change an unrelated thing for the PR so how about making another PR to handle this activation change?

Reply:
ok i'll do that